proto/kuksa/val/v2/types.proto

```diff
@@ -47,6 +47,24 @@ message Value {
   }
 }
 
+message SampleInterval {
+  int32 max_interval_ms = 1;
```

Comment:
If we keep this as separate structure, I'd remove the max to make this usable to possibly add a maximum interval (.i.e. minimum rate) in the `Filter` message ...
```suggestion
  int32 interval_ms = 1;
```